src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -786,6 +794,51 @@ internal void IncrementPasteOffset()
             this.currentPasteOffset = (this.currentPasteOffset + PASTE_OFFSET_STEP) % PASTE_OFFSET_MAX;
         }
 
+        internal void SetWorkspaceToState(PresetState state)
+        {
+            if (state == null)
+            {
+                Log("that state cannot be found");
+                return;
+            }
```

Comment:
This is not an accurate logging message.  This should either throw an `ArgumentNullException` or, if you want the no-op, log a message like `"Attempted to apply a PresetState that was null"`